ci/jobs/scripts/ssh-setup.sh

```diff
@@ -1,5 +1,2 @@
-#!/usr/bin/env bash
-set -euo pipefail
```

Comment:
If any command produces a non-zero return code, even in the middle of a piped sequence, then this will cause bash to abort execution and return that code. Also, if an uninitialized variable is used, then bash will abort execution. This basically lets us abide by the idea that "errors should never pass silently unless explicitly silenced."